parent/pom.xml

```diff
@@ -24,17 +24,18 @@
     </scm>
 
     <properties>
-        <astyanax.version>3.8.0-bv9</astyanax.version>
+        <astyanax.version>3.8.0-bv10</astyanax.version>
         <dropwizard-metrics-datadog.version>1.1.13</dropwizard-metrics-datadog.version>
-        <cassandra.version>2.2.4</cassandra.version>
+        <cassandra.version>2.2.19</cassandra.version>
```

Comment:
it makes sense to upgrade
``` <cassandra.driver.version>3.1.1</cassandra.driver.version>
```
to at least 3.1.4 version

Reply:
updated the version